### ISSUE TITLE
Unable to use the promises plugin migrating from cloudant npm package

### ISSUE THREAD

**Issue by pmbanugo** [cloudant/nodejs-cloudant#312]
I noticed I've been using a wrong package. I've swapped my code to use this library but the promises library seems to be failing. 

initialisation:

```
let cloudant = Cloudant({
  account: account,
  password: password,
  plugin: "promises"
});
```

then I call to retrieve the doc: `let doc = await db.get(appId);` It returns the wrong object. It doesn't throw an error but the value after await is a different object that seems to contains events/functions or streams.  
I've just switched library and didn't change any code. Using the `cloudant` library I installed `"request-promise-native": "1.0.5"` which I'm not sure if a different library is needed for `@cloudant/cloudant`

- Cloudant (nodejs-cloudant) 2.2.0
- Node version: 8.9.0

**Comment by smithsz** [comment]
Please see the [migration guide](https://github.com/cloudant/nodejs-cloudant/blob/master/api-migration.md).

If you still have problems then let us know. Please do include some sample code. It's near impossible to debug your problem without it.

Thanks.

**Comment by pmbanugo** [comment]
I noticed the difference when initializing the Cloudant object. The old `cloudant` package used `plugin` while this one used "plugins"

so my code switched from 
```
let cloudant = Cloudant({
  account: account,
  password: password,
  plugin: "promises"
});
```

To
```
let cloudant = Cloudant({
  account: account,
  password: password,
  plugins: "promises"
});
```

This is the only change I made to get it working well with the promise library. I'm not sure if there are anything more I need to know about migration. I've read your migration guide but it seems to be migration from different versions of `@cloudant/cloudant` package. I am migrating from `cloudant` to `@cloudant/cloudant`.
